### ISSUE TITLE
Why do we need '--metric period =200' for evaluation? 

### ISSUE THREAD

**Issue by HaoZhouGT** [CityBrainChallenge/KDDCup2021-CityBrainChallenge-starter-kit#32]
Hi can you guys explain the setting of --metric_period 200 for evaluation? Does it mean we don't care the delay or throughput of the network after 2000s? Why not using --metric_period 359 for the whole period?

**Comment by Kanstarry9T** [comment]
--metric_period=200 means that we currently evaluate your solution every 200 seconds for scoring. You can set this parameter for your own needs.

**Comment by HaoZhouGT** [comment]
> --metric_period=200 means that we currently evaluate your solution every 200 seconds for scoring. You can set this parameter for your own needs.

Hi Kan, do you imply we can change this setting? I assume this is not modifiable. Also I found only by adding --metric_period=200 we can get the same result as the server does.

**Comment by Kanstarry9T** [comment]
Yes, you can modify --metric_period, for example,  --metric_period=100 means that evaluate your solution locally every 100 second, but please be notified currently, the scoring system evaluate your solution every 200 seconds. Also if you have any further questions, please join our Q&A session 06/02/2021 9:00PM (Beijing time) or 9:00 AM (Atlanta, EDT) https://bluejeans.com/895107273/0765
